LocalMultiplayerAgent/MultiplayerServerManager.cs

```diff
@@ -163,6 +164,72 @@ private void ExtractAssets(string assetFileName, string targetFolder)
 
                 _systemOperations.ExtractToDirectory(assetFileName, targetFolder);
             }
+            else
+            {
+                _systemOperations.CreateDirectory(targetFolder);
+
+                ProcessStartInfo processStartInfo = Path.GetExtension(assetFileName).ToLowerInvariant() == Globals.ZipExtension
+                   ? GetProcessStartInfoForZip(assetFileName, targetFolder)
+                   : GetProcessStartInfoForTarOrGZip(assetFileName, targetFolder);
+
+                _logger.LogInformation($"Starting asset extraction with command arguments: {processStartInfo.Arguments}");
```

Comment:
this code is copy pasted from VmAgent right? We should just move it to a separate AssetExtraction class and use the same one in both places

Reply:
Thanks for the feedback Rakesh. Does it mean we just need to create separate file (for example) **AssetProcessStartInfo.cs** and put that under VmAgent.Core? Do you think there is a scenario in the future LocalMultiplayerAgent will use asset extraction code (defined in VmAgent) like AssetExtraction.cs, AssetExtractorFactory.cs, ZipExtracion.cs and etc... These files currently used when VmAgent download asset as stream but not sure if I need to move those files too.

Reply:
YEah I would call it Basic(or Simple)AssetExtractor.cs and put the ExtractAsset() method along with the private methods for GetProcessStartInfo in VmAgent.Core. We don't need the AssetExtractorFactory and other ones here yet. They will moved if and when we open source the rest of it. They are specific for streaming that's not used here.

Reply:
Makes sense. Thanks :)